src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -124,7 +140,9 @@ class DocumentStatisticsTrainingOnlyPolicy {
     using InputType                         = std::string;
     using InputTypeConstIterator            = std::string::const_iterator;
     using StringDecorator                   = std::function<std::string (InputTypeConstIterator begin, InputTypeConstIterator end)>;
-    using IndexMap                          = DocumentStatisticsAnnotationData::IndexMap;
+    using IndexMap                          = std::unordered_map<std::string, std::uint32_t>;
+    using FrequencyMap                      = IndexMap;
```

Comment:
It looks like this can be a private type.

Reply:
FrequencyAndIndexMap can be a private type, while the functions in anonymous space uses the public IndexMap and FrequencyMap. Would it be better to move the functions from anonymous space to DocumentStatisticsTrainingOnlyPolicy? or just use the std::unordered_map as the return type?

Reply:
If the type is only used in the anonymous namespace, it can be defined there.